src/content/time-tracking.mdx

```diff
@@ -0,0 +1,65 @@
+# Time Tracking Guide
+
+This tracking guide is not the same as general company’s rules. It relates only developer stuff and additional things which can be helpful for our work.
+For more projects/tasks details, please read our company tracking rules.
+
+## Where should I track my time?
+We are using a Harvest app to store tracked time. You can use a web / phone / desktop app or Chrome / Mozilla / Microsoft Edge extension [(look here)](https://www.getharvest.com/apps-and-integrations/directory#mobile-desktop). You can choose which one will be the best for you, to make it as easy as it’s possible & not disturb you while your daily stuff.
+
+
+## How should I track?
+
+### Project’s tasks
+It was intentionally described „Harvest app to store tracked time”, because it’s not always a place where you should add tasks.
+**To track any project’s task we are using a JIRA-Harvest connection**, which allows you to start and stop timer related to current task without your special involvement.
+The only thing you should do is to find a „Harvest Time Tracking by 42nd” in a task on JIRA and then connect your harvest to it. If you did it, you can just start timer directly from a task card and **it will be automatically visible in your harvest timesheet**, with proper project, task category and description with a link to the task added.
```

Comment:
```suggestion
The only thing you should do is to find the „Harvest Time Tracking by 42nd” in the task on JIRA and then connect your Harvest to it. When you're done, you can just start timer directly from the task's card and **it will be automatically visible in your harvest timesheet**, with proper project, task category and description and a link to the task added.
```